### ISSUE TITLE
IWKNavigationDelegate has two interface methods, only show one code completion method

### ISSUE THREAD

**Issue by dalexsoto** [mono/monodevelop#4216]
_From @LeeCenY on March 16, 2018 10:28_

### Steps to Reproduce

1.   mvWebView.NavigationDelegate = this;
2.   IWKNavigationDelegate
3.  decidePolicy
4.  decidePolicy

### Expected Behavior

[Export("webView:decidePolicyForNavigationResponse:decisionHandler:")]
[Export("webView:decidePolicyForNavigationAction:decisionHandler:")]

```
[Export("webView:decidePolicyForNavigationResponse:decisionHandler:")]
public void DecidePolicy(WKWebView webView, WKNavigationResponse navigationResponse, Action<WKNavigationResponsePolicy> decisionHandler)
{
   throw new System.NotImplementedException();
}
```
```
[Export("webView:decidePolicyForNavigationAction:decisionHandler:")]
public void DecidePolicy(WKWebView webView, WKNavigationAction navigationAction, Action<WKNavigationActionPolicy> decisionHandler)
{
    throw new System.NotImplementedException();
}
```
### Actual Behavior

[Export("webView:decidePolicyForNavigationResponse:decisionHandler:")]
[Export("webView:decidePolicyForNavigationResponse:decisionHandler:")]

```
[Export("webView:decidePolicyForNavigationResponse:decisionHandler:")]
public void DecidePolicy(WKWebView webView, WKNavigationResponse navigationResponse, Action<WKNavigationResponsePolicy> decisionHandler)
{
   throw new System.NotImplementedException();
}
```
```
[Export("webView:decidePolicyForNavigationResponse:decisionHandler:")]
public void DecidePolicy(WKWebView webView, WKNavigationResponse navigationResponse, Action<WKNavigationResponsePolicy> decisionHandler)
{
   throw new System.NotImplementedException();
}
```

### Environment

```
=== Visual Studio Community 2017 for Mac ===

Version 7.4 (build 1033)
Installation UUID: 891dc29f-4210-4ba9-a91b-9564b6fea41d
Runtime:
	Mono 5.8.0.129 (2017-10/835a98d8559) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)

	Package version: 508000129

=== NuGet ===

版本: 4.3.1.4445

=== .NET Core ===

运行时: /usr/local/share/dotnet/dotnet
运行时版本:
	2.0.5
	2.0.0
SDK: /usr/local/share/dotnet/sdk/2.1.4/Sdks
SDK 版本:
	2.1.4
	2.0.0
MSBuild SDK: /Library/Frameworks/Mono.framework/Versions/5.8.0/lib/mono/msbuild/15.0/bin/Sdks

=== Xamarin.Profiler ===

版本: 1.6.1
位置:/Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Apple Developer Tools ===

Xcode 9.2 (13772)
Build 9C40b

=== Xamarin.Android ===

版本: 8.2.0.15 (Visual Studio Community)
Android SDK: /Users/ttlgzmac6/Library/Developer/Xamarin/android-sdk-macosx
	受支持的 Android 版本:
		2.3   (API 级别 10)
		4.0.3 (API 级别 15)
		4.4   (API 级别 19)
		5.0   (API 级别 21)
		6.0   (API 级别 23)
		7.0   (API 级别 24)
		7.1   (API 级别 25)
		8.1   (API 级别 27)

SDK 工具版本: 25.2.5
SDK 平台工具版本: 27.0.1
SDK 生成工具版本: 27.0.0

Java SDK: /usr
java version "1.8.0_131"
Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)

Android Designer EPL 代码在此处提供:
https://github.com/xamarin/AndroidDesigner.EPL

=== Xamarin.Mac ===

Version: 4.2.0.20 (Visual Studio Community)

=== Xamarin.iOS ===

Version: 11.8.0.20 (Visual Studio Community)
Hash: 1c6f300e
Branch: d15-6
Build date: 2018-02-07 23:42:40-0500

=== Xamarin Inspector ===

Version: 1.4.0
Hash: b3f92f9
Branch: master
Build date: Fri, 19 Jan 2018 22:00:34 GMT
Client compatibility: 1

=== Build Information ===

Release ID: 704001033
Git revision: 053991068f3c88497d3e5cc52fe7fdf57c28b62f
Build date: 2018-02-28 05:02:28-05
Xamarin addins: cc3283953e0af0b8bfab1bdfdf847d9e403c4606
Build lane: monodevelop-lion-d15-6

=== Operating System ===

Mac OS X 10.13.0
Darwin 17.0.0 Darwin Kernel Version 17.0.0
    Thu Aug 24 21:48:19 PDT 2017
    root:xnu-4570.1.46~2/RELEASE_X86_64 x86_64

=== Enabled user installed addins ===

Internet of Things (IoT) development (Preview) 7.1


```

### Example Project (If Possible)

_Copied from original issue: xamarin/xamarin-macios#3762_
> VS bug [#585126](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/585126)

**Comment by dalexsoto** [comment]
I am able to reproduce this issue, it seems that optional Delegate members are not getting picked up by the IDE autocompletion

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #4387 merged by: Oleg Demchenko
Author: mono
Commit: 3796ac22b5ff35651540c96b170f6e681de2b19f (xamarin/monodevelop)
